bin/polyply

```diff
@@ -110,6 +91,49 @@ def main():
 
     parser_gen_coords.set_defaults(func=gen_coords)
 
+    #============================================================================
+    #           Input Arguments for the sequence generation tool
+    #============================================================================
+
+    parser_gen_seq.add_argument('-name', required=True, type=str, dest="name",
+                                   help="name of the final molecule")
+
+    file_group = parser_gen_seq.add_argument_group('Input and output files')
+    file_group.add_argument('-f', dest='ffpath', required=False, type=Path,
+                            help='force-field files (.ff|.itp)', nargs='*')
+    file_group.add_argument('-o', dest='outpath', type=Path,
+                            help='output file (.json)')
+
+
+    macro_group = parser_gen_seq.add_argument_group('Definitions of macros')
+    macro_group.add_argument('-from_string', dest='macros', nargs='+', type=str,
+                            help=("Define small polymer fragments: "
+                                  "the format is <tag>:<#blocks>:<#branches>:<residues> "
+                                  "where residues has the format <resname-probability>. "
+                                  "Examples are linear PEG of length 10 <A:10:1:PEG-1.0> "
+                                  "or a random copolymer of PS-PEG <A:10:1:PS-0.5,PEG-0.5>. "
+                                  "But we can also generate branched polymer with 3 generations "
+                                  "using <A:3:3:NR3:1>."))
+    macro_group.add_argument('-from_file', dest='macros', nargs='+', type=str,
+                            help="Use a molecule defined in an input file as fragment. The format "
+                                 "is <tag:molecule name>")
+    seq_group = parser_gen_seq.add_argument_group('Definition of sequence')
+    seq_group.add_argument('-seq', dest='seq', nargs='+', type=str,
+                            help="Define the sequence order in which to combine macros."
+                                 "The format is <macro_tag, macro_tag, ...>.")
```

Comment:
With spaces? Could use an example

Reply:
yes with spaces just like A B C D or whatever the blocks are called.

Reply:
`A B C D` or `A, B, C, D` or `A,B,C,D`?